### DISCUSSION TITLE
Shortcut for CREATE / ADJUST: Insert new subtitle at video pos (no text box focus) + minus gap

### DISCUSSION THREAD

**Discussion by Fhining** [SubtitleEdit/subtitleedit#8958] [Ideas]
Hello! Hope everyone’s doing well!

1- I've been trying to use the shortcut: Insert new subtitle at video position (no text box focus), but it would be great if, when the subtitle you're adding overlaps with the previous one, it automatically creates the minimum gap (2 frames in Netflix's case), adjusting the end time of the previous subtitle to match the start of the new one. I've checked other options but haven't found one that exactly meets this need.

It would be a new shortcut and not adding minus gap to this one. As, sometimes, if you're using SE for dubbing, for example, overlapping subtitles would be a must. So I'm proposing a new shortcut with the minus gap added to this one.

2- Also, I've been wondering if there’s a way to adjust the default duration of a subtitle when adding a new one. It would be helpful if the default minimum duration could be shortened by a frame or two, so the initial subtitle block isn't overly long.

Thank you so much in advance!

**Comment by Fhining** [comment]
Hello @niksedk! Hope you're having a great day!

Sorry for mentioning you, but I wanted to ask you if the second point it's possible:

2- If there’s a way to adjust the default duration of a subtitle when adding a new one. It would be helpful if the default minimum duration could be shortened by a frame or two, so the initial subtitle block isn't overly long.

I can't set the minimum default duration to less than 100ms. It would be really helpful if this could be customizable starting from 1 instead of 100ms.

![image](https://github.com/user-attachments/assets/9586d490-f3d4-4f41-8498-aa578d62f57f)


Thank you so much in advance! 😊

**Comment by darnn** [comment]
I would second this and add that it would be useful to me to be able to insert subtitles that are zero frames long, if I'm editing a file that has no timecodes to begin with.
